content/SCALETutorials/DataProtection/PeriodicSnapshotTasksSCALE.md

```diff
@@ -93,3 +93,16 @@ For example, you have two schedules created where one schedule takes a snapshot
 Each has an hourly snapshot taken.
 After a week, snapshots created at *01.00* through *23.00* get deleted, but you keep snapshots timed at *00.00* because they are necessary for the second periodic task. 
 These snapshots get destroyed at the end of 3 years.
+
+### Configuring Snapshot Granularity
```

Comment:
Why would a user want to configure this? We should say why.
Second, you don't actually tell them how to configure it, just provide information about it.
If you want to change this to and "about snapshot granularity" header that fits the information contained in this section.
Recommend either changing the header, or adding instructions on how to configure it.